lib/views/pages/intro/constantly_moving_lottie.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:flutter/material.dart';
+import 'package:lottie/lottie.dart';
+
+class ConstantlyMoving extends StatefulWidget {
+  const ConstantlyMoving({super.key, required this.asset});
```

Comment:
size は 外から指定できた方が調整しやすそう。

```suggestion
const ConstantlyMoving({super.key, this.size = 380});
final int size;
...
return SizedBox(width: size, height: size,...)
...
```